core/metacling/src/TCling.cxx

```diff
@@ -626,6 +626,16 @@ extern "C" int TCling__AutoLoadCallback(const char* className)
    return ((TCling*)gCling)->AutoLoad(className);
 }
 
+extern "C" int TCling__AutoLoadLibraries(std::string ModuleName)
+{
+   std::string filename = "lib" + ModuleName + ".so";
+   // If already loaded, return 0
+   if (((TCling*)gCling)->IsLoaded(filename.c_str()))
```

Comment:
I'm surprised `TCling::Load()` isn't already doing that. I know `TSystem::Load()` has a different return value depending on whether the library was already loaded before or not; is that not coming from TCling? Should we then maybe use `TSystem::Load()` here?